arches_lingo/src/arches_lingo/pages/SchemeList.vue

```diff
@@ -30,11 +31,11 @@ onMounted(async () => {
         });
     }
     schemes.value.unshift({
-        resourceinstanceid: "placeholder-id",
+        resourceinstanceid: NEW,
         descriptors: {
             en: {
                 name: "Create a new scheme",
```

Comment:
Is hardcoding of this value desirable?  Seems like if new languages get added we'd have to revisit these descriptors.